client/src/main/java/io/lionweb/client/inmemory/RepositoryData.java

```diff
@@ -84,10 +98,29 @@ private void removeNode(String removeNodeId) {
           removeNode(child);
         }
       }
+      for (String child : serializedClassifierInstance.getAnnotations()) {
```

Comment:
(naming) `annotationId` instead of `child`?

Reply:
Right!